{"name": "adding new address", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//a[@data-csa-c-content-id='nav_ya_signin']\"}\n  (Session info: chrome=116.0.5845.111); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n\tGetHandleVerifier [0x00007FF7478A52A2+57122]\n\t(No symbol) [0x00007FF74781EA92]\n\t(No symbol) [0x00007FF7476EE3AB]\n\t(No symbol) [0x00007FF747727D3E]\n\t(No symbol) [0x00007FF747727E2C]\n\t(No symbol) [0x00007FF747760B67]\n\t(No symbol) [0x00007FF74774701F]\n\t(No symbol) [0x00007FF74775EB82]\n\t(No symbol) [0x00007FF747746DB3]\n\t(No symbol) [0x00007FF74771D2B1]\n\t(No symbol) [0x00007FF74771E494]\n\tGetHandleVerifier [0x00007FF747B4EF82+2849794]\n\tGetHandleVerifier [0x00007FF747BA1D24+3189156]\n\tGetHandleVerifier [0x00007FF747B9ACAF+3160367]\n\tGetHandleVerifier [0x00007FF747936D06+653702]\n\t(No symbol) [0x00007FF74782A208]\n\t(No symbol) [0x00007FF7478262C4]\n\t(No symbol) [0x00007FF7478263F6]\n\t(No symbol) [0x00007FF7478167A3]\n\tBaseThreadInitThunk [0x00007FFE43FE7614+20]\n\tRtlUserThreadStart [0x00007FFE45E826B1+33]\n\n", "trace": "  File \"C:\\Users\\lmallikarjuna\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Users\\lmallikarjuna\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\add_address.py\", line 13, in step_impl\n    context.login_page = context.homepage.click_on_my_account()\n                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\POC-\\features\\pages\\home_page.py\", line 17, in click_on_my_account\n    self.click_on_element(\"my_account_option_xpath\", self.my_account_option_xpath)\n  File \"D:\\POC-\\features\\pages\\BasePage.py\", line 32, in click_on_element\n    element = self.get_element(locator_type, locator_value)\n              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\POC-\\features\\pages\\BasePage.py\", line 46, in get_element\n    element = self.driver.find_element(By.XPATH, locator_value)\n              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\lmallikarjuna\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 739, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\lmallikarjuna\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 345, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\lmallikarjuna\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "steps": [{"name": "Given I logged into my account and got navigated to Homepage", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//a[@data-csa-c-content-id='nav_ya_signin']\"}\n  (Session info: chrome=116.0.5845.111); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n\tGetHandleVerifier [0x00007FF7478A52A2+57122]\n\t(No symbol) [0x00007FF74781EA92]\n\t(No symbol) [0x00007FF7476EE3AB]\n\t(No symbol) [0x00007FF747727D3E]\n\t(No symbol) [0x00007FF747727E2C]\n\t(No symbol) [0x00007FF747760B67]\n\t(No symbol) [0x00007FF74774701F]\n\t(No symbol) [0x00007FF74775EB82]\n\t(No symbol) [0x00007FF747746DB3]\n\t(No symbol) [0x00007FF74771D2B1]\n\t(No symbol) [0x00007FF74771E494]\n\tGetHandleVerifier [0x00007FF747B4EF82+2849794]\n\tGetHandleVerifier [0x00007FF747BA1D24+3189156]\n\tGetHandleVerifier [0x00007FF747B9ACAF+3160367]\n\tGetHandleVerifier [0x00007FF747936D06+653702]\n\t(No symbol) [0x00007FF74782A208]\n\t(No symbol) [0x00007FF7478262C4]\n\t(No symbol) [0x00007FF7478263F6]\n\t(No symbol) [0x00007FF7478167A3]\n\tBaseThreadInitThunk [0x00007FFE43FE7614+20]\n\tRtlUserThreadStart [0x00007FFE45E826B1+33]\n\n", "trace": "  File \"C:\\Users\\lmallikarjuna\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Users\\lmallikarjuna\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\add_address.py\", line 13, in step_impl\n    context.login_page = context.homepage.click_on_my_account()\n                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\POC-\\features\\pages\\home_page.py\", line 17, in click_on_my_account\n    self.click_on_element(\"my_account_option_xpath\", self.my_account_option_xpath)\n  File \"D:\\POC-\\features\\pages\\BasePage.py\", line 32, in click_on_element\n    element = self.get_element(locator_type, locator_value)\n              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\POC-\\features\\pages\\BasePage.py\", line 46, in get_element\n    element = self.driver.find_element(By.XPATH, locator_value)\n              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\lmallikarjuna\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 739, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\lmallikarjuna\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 345, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\lmallikarjuna\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "attachments": [{"name": "screenshot", "source": "af24d91c-31d7-42b0-a75c-608c6070729b-attachment.png", "type": "image/png"}], "start": 1693397298384, "stop": 1693397298562}, {"name": "When I click on my account and go to address option", "status": "skipped", "start": 1693397301253, "stop": 1693397301253}, {"name": "And click on add a new address", "status": "skipped", "start": 1693397301253, "stop": 1693397301253}, {"name": "Then entered all the details and added it", "status": "skipped", "start": 1693397301253, "stop": 1693397301253}], "start": 1693397290304, "stop": 1693397301253, "uuid": "a71749f8-e5fa-4a7f-990c-1aa0c047c5de", "historyId": "12dd1a5d88f4de8a6e7a648477971cb2", "fullName": "print transactions history: adding new address", "labels": [{"name": "severity", "value": "normal"}, {"name": "tag", "value": "address"}, {"name": "feature", "value": "print transactions history"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}